packages/test/test-utils/src/TestSummaryUtils.ts

```diff
@@ -108,51 +103,28 @@ export async function createSummarizerFromFactory(
 	return createSummarizerCore(container, loader, summaryVersion);
 }
 
-/**
- * Creates a summarizer client from the given container and returns the summarizer client's IContainer and ISummarizer.
- * The ISummarizer can be used to generate on-demand summaries. The IContainer can be used to fetch data stores, etc.
- */
-export async function createSummarizer(
-	provider: ITestObjectProvider,
-	container: IContainer,
-	summaryVersion?: string,
-	gcOptions?: IGCRuntimeOptions,
-	configProvider: IConfigProviderBase = mockConfigProvider(),
-	logger?: ITelemetryBaseLogger,
-): Promise<{ container: IContainer; summarizer: ISummarizer }> {
-	const testContainerConfig: ITestContainerConfig = {
-		runtimeOptions: {
-			summaryOptions: defaultSummaryOptions,
-			gcOptions,
-		},
-		loaderProps: { configProvider, logger },
-	};
-	const loader = provider.makeTestLoader(testContainerConfig);
-	return createSummarizerCore(container, loader, summaryVersion);
-}
-
 /**
  * Creates a summarizer client from the given container and returns the summarizer client's IContainer and ISummarizer.
  * The ISummarizer can be used to generate on-demand summaries. The IContainer can be used to fetch data stores, etc.
  *
  * Can pass in a test config provider to enable/disable features.
  */
-export async function createSummarizerWithTestConfig(
+export async function createSummarizer(
 	provider: ITestObjectProvider,
 	container: IContainer,
-	config: ITestContainerConfig,
+	config?: ITestContainerConfig,
 	summaryVersion?: string,
 	logger?: ITelemetryBaseLogger,
```

Comment:
logger is part of `ITestContainerConfig` too and can be removed as a separate param.